javascript/code-challenges/challenges-15.test.js

```diff
@@ -0,0 +1,266 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named screenForNames that takes in an array of strings and uses Regex to create a new array of only those strings that match the following rules:
+
+* the name must begin with Mr., Mrs., Ms., Dr. followed by a space
+* the name must contain only letter characters (white spaces are ok)
+
+------------------------------------------------------------------------------------------------ */
+
+const screenForNames = (arr) => {
+  // Solution code here...
+  let newarr = arr.filter(name => /^((Mr.)|(Mrs.)|(Ms.)|(Dr.))\s\w+$/.test(name));
+  return newarr;
+
+};
```

Comment:
This function passes the test provided but isn't quite restrictive enough. The period is a token telling you to match any single character, meaning something like "Dre Young" or "Dr! Young" would pass. You need to add a backslash to make it look for a literal period, e.g. `/Dr\./`